coursedata/texts/en.yaml

```
@@ -52,6 +52,13 @@ ui:
     not_enrolled: "Looks like you are not in this class!"
     retrieve_class: "Only teachers can retrieve classes"
 ClientErrorMessages:
+    Emoji_turtle: "🐢"
```

Great you are working on this @fRedelaar! However, maybe we can make a dictionary here, that would look a bit nicer? Something like:

Client
```suggestion
    ClientErrorMessages:
        turtle: "🐢",
        rock: "✂️, 📜, 🗻"
```

Then in ts we can look up a list with the name of the adventure, simplifying the code too? (see second suggestion)